content/patterns/patterns.html

```diff
@@ -11,367 +11,433 @@
     <link rel="stylesheet" href="../shared/css/core.css">
     <script src="../shared/js/highlight.pack.js"></script>
     <script src="../shared/js/app.js"></script>
+    <script src="../shared/js/filterBySearch.js"></script>
     <script data-skipto="colorTheme:aria; displayOption:popup; containerElement:div" src="../shared/js/skipto.js"></script>
+    <script>
+      window.onload = function() {
+        document.getElementById('listViewButton').addEventListener('click', function() {
+          updateView('list');
+        });
+        document.getElementById('tilesViewButton').addEventListener('click', function() {
+          updateView('tiles');
+        });
+      }
+  
+      function updateView(view) {
```

Comment:
For reviewers, this may be an anti-pattern. I can use the `aria.X.utilityMethod()` pattern from `filterBySearch.js` if preferred. I took this route to avoid complicating the domain of that file or adding another script for such a small task but I can see how this is divergent

Reply:
Thanks for acknowledging and calling this out. My immediate reaction would indeed be to keep it in the `filterBySearch.js` utility.

Reply:
Agreed! Addressed in [56cfb28](https://github.com/w3c/aria-practices/pull/3043/commits/56cfb284e6a26dccc843b2becb28ac4add2ef7c8)